File: src/main/java/com/cuupa/dms/ui/LocalDateTimeFormatter.java
package com.cuupa.dms.ui;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class LocalDateTimeFormatter {

    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy - HH:mm");

    public String format(LocalDateTime localDateTime) {
        return localDateTime.format(formatter);
    }
}

File: src/main/java/com/cuupa/dms/ui/LocalDateTimeFormatter.kt
package com.cuupa.dms.ui

import java.time.LocalDateTime
import java.time.format.DateTimeFormatter

class LocalDateTimeFormatter {
    fun format(localDateTime: LocalDateTime): String {
        return localDateTime.format(formatter)
    }

    companion object {
        private val formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy - HH:mm")
    }
}